### PR TITLE
Some updates to quagga package

### PR DESCRIPTION
**Pull request by mimugmail**
Hi,

this is my first pull. 

Added the zebra.conf since it is reqiured to start quagga itself. 
Also fixed some minor typos.

### CODE REVIEW

File: net/quagga/src/opnsense/service/templates/OPNsense/Quagga/zebra.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+!
+!
+!
+{% if helpers.exists('OPNsense.quagga.ospf.interfaces.interface') %}
+{%   for interface in helpers.toList('OPNsense.quagga.ospf.interfaces.interface') %}
+{%     if interface.enabled == '1' %}
+interface {{ interface.interfacename }}
+{%     endif %}
+{%   endfor %}
+{% endif %}
```
Comment (fabianfrz) [inline net/quagga/src/opnsense/service/templates/OPNsense/Quagga/zebra.conf]: There is one "!" between the interfaces and I think it is better to omit this settings as long as they are unused. Also Zebra should not use the `OSPF` settings. I think it is also valid to have a configuration file without interfaces inside.

Comment (mimugmail) [inline net/quagga/src/opnsense/service/templates/OPNsense/Quagga/zebra.conf]: Indeed, I copied the file from ospf and forgot to remove these parts!

**Comment by fabianfrz** [comment]
@mimugmail, now it looks ok but @fichtner needs to create a squashed commit (merging all commits into a single one) just for your information.

@fichtner We probably should bump the version number to 0.0.2 ;)

**Comment by fichtner** [comment]
@fabianfrz @mimugmail thanks guys, all merged!

FYI https://github.com/opnsense/plugins/commit/a99da6d6

**Pull request merged**